### PR TITLE
[#1] migrate utilities

### PR DESCRIPTION
**Pull request by MrDmitry**
Most of the pull request is straight migration of utilities from Unicorn repo. The rest is configuration releated.

### CODE REVIEW

File: MuleUtilities/MuleVersionInfo.rc.in [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#include <Windows.h>
```
Comment (ilia-glushchenko) [inline MuleUtilities/MuleVersionInfo.rc.in:9]: Copyrights?

**Review by ilia-glushchenko — CHANGES_REQUESTED** [review]

File: MuleUtilities/MuleVersionInfo.rc.in (hunk 1 above)
Comment (MrDmitry) [reply MuleUtilities/MuleVersionInfo.rc.in:9]: Done

**Comment by ilia-glushchenko** [comment]
Adding issue indices to the commit messages probably going to be a good thing as well.

**Comment by MrDmitry** [comment]
Done

**Review by ilia-glushchenko — APPROVED** [review]

**Review by krupitskas — APPROVED** [review]

**Pull request merged**